### PR TITLE
feat: add standalone install feedback component

### PR DESCRIPTION
**Pull request by roadlittledawn**
## Description

Moves code for install feedback to standalone component and supports using it in MDX files.

Also adds handlers for translation and when local plugins convert MDX to html

Need to verify that the JSON for the MDX file i added the component to successfully removes the component. I feel pretty good about the i18n handler.

## Related issues / PRs
Closes https://issues.newrelic.com/browse/NR-48549

### CODE REVIEW

**Comment by roadlittledawn** [comment]
note: i had opened the first PR and accidentally used `develop`. i then edited the PR to update the base branch to the feature branch. that created two separate gatsby cloud builds. i put up a 2nd PR under a different branch name and pointed it at feature branch. caused some weird gatsby cloud issues where i couldn't cancel the develop build. so i created a new branch, made changes anew, and pushed that up with base of feature branch, and then i was able to cancel previous builds.

**Comment by roadlittledawn** [comment]
will look into making the thumb buttons more responsive

File: src/components/InstallFeedback.js [hunk 1]
```diff
@@ -0,0 +1,225 @@
...
+        border: 1px solid #afe2e3;
+        background-color: rgba(175, 226, 227, 0.05);
+      `}
+    >
+      <div
+        css={css`
+          display: flex;
+          flex-direction: row;
+
+          @media screen and (max-width: 1200px) {
```
Comment (clarkmcadoo) [inline src/components/InstallFeedback.js:47]:
```suggestion
          @media screen and (max-width: 1500px) {
```

will get those thumbs up/down looking right

File: src/components/InstallFeedback.js [hunk 2]
```diff
@@ -0,0 +1,225 @@
...
+            flex-direction: column;
+          }
+        `}
+      >
+        <div
+          css={css`
+            margin-right: 10px;
+
+            @media screen and (max-width: 1200px) {
```
Comment (clarkmcadoo) [inline src/components/InstallFeedback.js:56]:
```suggestion
          @media screen and (max-width: 1500px) {
```

here too

**Review by clarkmcadoo — APPROVED** [review]

**Pull request merged**